puzzle-utils/encoders/caesar.py

```diff
@@ -0,0 +1,35 @@
+# All input should be uppercased
```

Comment:
Import this file in `puzzlehunt-utils.py` so that we can use the functions in it in interactive mode